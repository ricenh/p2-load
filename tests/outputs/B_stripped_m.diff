--- outputs/B_stripped_m.txt	2023-10-10 12:39:48.039338223 -0400
+++ expected/B_stripped_m.txt	2023-09-19 13:29:21.000000000 -0400
@@ -0,0 +1,5 @@
+Contents of memory from 0100 to 0115:
+  0100  30 f3 0f 00 00 00 20 31  40 13 fd ff ff ff 60 31
+  0110  70 08 01 00 00
+Contents of memory from 0200 to 020b:
+  0200  aa bb cc dd 00 00 00 dd  00 00 00
