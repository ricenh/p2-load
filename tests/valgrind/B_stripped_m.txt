==1045766== Memcheck, a memory error detector
==1045766== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1045766== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1045766== Command: ../y86 -m inputs/stripped.o
==1045766== 
../y86: invalid option -- 'm'
Usage: ../y86 <option(s)> mini-elf-file
 Options are:
  -h      Display usage
  -H      Show the Mini-ELF header
==1045766== 
==1045766== HEAP SUMMARY:
==1045766==     in use at exit: 0 bytes in 0 blocks
==1045766==   total heap usage: 1 allocs, 1 frees, 8,192 bytes allocated
==1045766== 
==1045766== All heap blocks were freed -- no leaks are possible
==1045766== 
==1045766== For lists of detected and suppressed errors, rerun with: -s
==1045766== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
