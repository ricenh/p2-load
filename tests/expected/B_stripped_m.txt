Contents of memory from 0100 to 0115:
  0100  30 f3 0f 00 00 00 20 31  40 13 fd ff ff ff 60 31
  0110  70 08 01 00 00
Contents of memory from 0200 to 020b:
  0200  aa bb cc dd 00 00 00 dd  00 00 00
